apps/pie-docs/src/foundations/motion/overview/motion-principles.json

```diff
@@ -0,0 +1,32 @@
+{
+    "items": [
+        {
+            "headingText": "Purposeful",
+            "imgSrc": "../../../assets/img/foundations/motion/principles/purposeful.svg",
+            "imgSrcNarrow": "../../../assets/img/foundations/motion/principles/purposeful_narrow.svg",
+            "text": "Motion always feels intentional and appropriate for the context it’s being used in. It aligns with the user's goals and commands focus without causing unnecessary distractions.",
+            "alt": "Bright, abstract illustration depicting purposeful."
+        },
+        {
+            "headingText": "Smooth",
+            "imgSrc": "../../../assets/img/foundations/motion/principles/smooth.svg",
+            "imgSrcNarrow": "../../../assets/img/foundations/motion/principles/smooth_narrow.svg",
+            "text": "Motion mimics real-world physics to always look and feel fluid, natural and seamless. Motion is predictable and tells a story without feeling jarring or out of place in our products.",
+            "alt": "Bright, abstract illustration depicting smoothness."
+        },
+        {
+            "headingText": "Responsive",
+            "imgSrc": "../../../assets/img/foundations/motion/principles/responsive.svg",
+            "imgSrcNarrow": "../../../assets/img/foundations/motion/principles/responsive_narrow.svg",
+            "text": "Motion informs and guides our users by providing visual feedback and driving a user's attention. It reinforces hierarchy between elements and is timely to avoid feeling intrusive or sluggish.",
+            "alt": "Bright, abstract illustration depicting responsiveness."
+        },
+        {
+            "headingText": "Delightful",
+            "imgSrc": "../../../assets/img/foundations/motion/principles/delightful.svg",
+            "imgSrcNarrow": "../../../assets/img/foundations/motion/principles/delightful_narrow.svg",
+            "text": "Motion aims to delight our users and express our brand personality through enhancing the experience of our products and by using playful movements, where appropriate, to make our products more enjoyable to use.",
```

Comment:
As per the figma, it should be instead:

Motion in our products aim to delight our users and express our brand personality through enhancing the experience of our products and by using playful movements, where appropriate, to make our products more enjoyable to use.

Reply:
hm, seems it's been changed in Figma, because I copied text from there. 
I will update, thank you